hasher-matcher-actioner/webapp/src/components/settings/ActionRuleFormColumns.tsx

```diff
@@ -213,18 +213,15 @@ export default function ActionRuleFormColumns({
       <td>
         <Form.Label>
           Action
-          <span hidden={!showErrors || actionRule.action !== '0'}>
-            {' '}
-            (required)
-          </span>
+          <span hidden={!showErrors || !!actionRule.action}> (required)</span>
         </Form.Label>
         <Form.Control
           as="select"
           required
           value={actionRule.action}
           onChange={e => onChange('action_id', e.target.value)}
```

Comment:
unsure: feels like this might now break something (or at least result in some confusion) based on the fact that it isn't called 'action_id' anymore?

Reply:
Changed to action name